sql-statements/sql-statement-alter-table-compact.md

```diff
@@ -52,9 +52,28 @@ ALTER TABLE employees SET TIFLASH REPLICA 2;
 ALTER TABLE employees COMPACT TIFLASH REPLICA;
 ```
 
+### 对分区表中个别分区的 TiFlash 副本进行数据整理
+假设目前有一张 `employees` 表具有 4 个分区，且具有 2 个 TiFlash 副本：
```

Comment:
```suggestion
### 对分区表中个别分区的 TiFlash 副本进行数据整理

假设目前有一张 `employees` 表具有 4 个分区，且具有 2 个 TiFlash 副本：
```

Reply:
BTW I would recommend to install the Prettier plugin in VsCode, so that you can have auto format on save (for Markdown and some other langs), which is very handy.